code/game/machinery/medical_kiosk.dm

```diff
@@ -0,0 +1,345 @@
+	else if((brain_loss) >= 1)
+		brain_status = "Mild brain damage detected."  //You may have a miiiild case of severe brain damage.
+
+	if(altPatient.radiation >=1000)  //
+		rad_status = "Patient is suffering from extreme radiation poisoning. Suggested treatment: Isolation of patient, followed by repeated dosages of Pentetic Acid."
+	else if(altPatient.radiation >= 500)
+		rad_status = "Patient is suffering from alarming radiation poisoning. Suggested treatment: Heavy use of showers and decontamination of clothing. Take Pentetic Acid or Potassium Iodine."
+	else if(altPatient.radiation >= 100)
+		rad_status = "Patient has moderate radioactive signatures. Keep under showers until symptoms subside."
+
+	if(pandemonium == TRUE)
+		chaos_modifier = 1
+	else if (user.hallucinating())
+		chaos_modifier = 0.3
+
+	data["kiosk_cost"] = active_price + (chaos_modifier * (rand(1,25)))
+	data["patient_name"] = patient_name
+	data["patient_health"] = round(((total_health - (chaos_modifier * (rand(1,50)))) / max_health) * 100, 0.001)
+	data["brute_health"] = round(brute_loss+(chaos_modifier * (rand(1,30))),0.001)		//To break this down for easy reading, all health values are rounded to the .001 place
+	data["burn_health"] = round(fire_loss+(chaos_modifier * (rand(1,30))),0.001)		//then a random number is added, which is multiplied by chaos modifier.
+	data["toxin_health"] = round(tox_loss+(chaos_modifier * (rand(1,30))),0.001)		//That allows for a weaker version of the affect to be applied while hallucinating as opposed to emagged.
+	data["suffocation_health"] = round(oxy_loss+(chaos_modifier * (rand(1,30))),0.001)	//It's not the cleanest but it does make for a colorful window.
+	data["clone_health"] = round(clone_loss+(chaos_modifier * (rand(1,30))),0.001)
+	data["brain_health"] = brain_status
+	data["brain_damage"] = brain_loss+(chaos_modifier * (rand(1,30)))
+	data["patient_status"] = patient_status
+	data["rad_value"] = rad_value+(chaos_modifier * (rand(1,500)))
+	data["rad_status"] = rad_status
+	data["trauma_status"] = trauma_status
+	data["patient_illness"] = sickness
+	data["illness_info"] = sickness_data
+	data["bleed_status"] = bleed_status
+	data["blood_levels"] = blood_percent - (chaos_modifier * (rand(1,35)))
+	data["blood_status"] = blood_status
+	data["chemical_list"] = chemical_list
+	data["overdose_list"] = overdose_list
+	data["addict_list"] = addict_list
+	data["hallucinating_status"] = hallucination_status
+
+	data["active_status_1"] = scan_active_1 // General Scan Check
+	data["active_status_2"] = scan_active_2	// Symptom Scan Check
+	data["active_status_3"] = scan_active_3	// Radio-Neuro Scan Check
+	data["active_status_4"] = scan_active_4	// Radio-Neuro Scan Check
+	return data
+
+/obj/machinery/medical_kiosk/ui_act(action,active)
+	if(..())
+		return
+	switch(action)
+		if("beginScan_1")
+			inuse()
+			if(paying_customer == TRUE)
+				scan_active_1 = TRUE
+				paying_customer = FALSE
+		if("beginScan_2")
```

Comment:
Weird casing, but OK.
We tend to go with snake_case in SS13, rather than this camelCase you're using